types/site/_data/postsData.d.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare global {
+  export interface AuthorsItem extends VirtualCollectionItem, AuthorsDataItem {}
+
+  export interface Authors {
+    [key: string]: AuthorsItem;
+  }
+
+  export interface FrontMatterData {
```

Comment:
Don't we define this already somewhere—in "_types/data.d.ts"?

Reply:
We do not, this is a new object. Though `EleventyData` in `types/11ty/data.d.ts` is similar.

Reply:
I've answered my own question: because `FrontMatterData` is read via the gray-matter approach using direct file access, completely avoiding slow 11ty.